tests-stress/locust/locust-notifications.py

```diff
@@ -0,0 +1,24 @@
+import os
+
+from dotenv import load_dotenv
+from locust import HttpUser, constant_pacing, task
+
+load_dotenv()
+API_KEY = os.getenv("TEST_API_KEY")
+
+
+class NotifyApiUser(HttpUser):
+
+    wait_time = constant_pacing(60)
+    host = "https://api.staging.notification.cdssandbox.xyz"
+
+    @task
+    def send_notifications(self):
+        headers = {"Authorization": API_KEY, "Content-Type": "application/json"}
```

Comment:
This is an authorization header atm because you have to include the auth scheme. Can you update the name here or put the scheme in the codebase and update values in LastPass?

Reply:
I believe you don't need to specify the content type if you use `json=json` with requests, if you want to simplify